tests/utils_tests.py

```diff
@@ -352,6 +357,59 @@ def heat_to_discharge_test(solution, results):
         )
 
 
+def electric_power_conservation_test(solution, results):
+    """
+    Test to check if the electric power is conserved at every timestep
```

Comment:
Suggestion? add some high level detail bullet points stating what is checked

Reply:
removed unnecessary in a test.